ui/src/app/dashboard/views/DiscoveryAddForm/DiscoveryAddFormFields/DiscoveryAddFormFields.tsx

```diff
@@ -48,7 +48,7 @@ const DiscoveryAddFormFields = ({
   const isDevice = values.type === DeviceType.DEVICE;
   const isInterface = values.type === DeviceType.INTERFACE;
   // Only include static when the discovery has a subnet.
-  const includeStatic = discovery.subnet || discovery.subnet === 0;
+  const includeStatic = discovery.subnet !== null;
```

Comment:
Was this just a cleanup or was there an issue there? I wonder if the previous version was a bit safer if the API were to return something like `""`.

Reply:
Oh I was just making sure `includeStatic` was a boolean before passing it to the component, because before it was `number | boolean`. I suppose `!!discovery.subnet || discovery.subnet === 0` would be the safest option then